frontend/apps/pwa/src/components/RecordPicker.tsx

```diff
@@ -7,8 +7,8 @@ import { selectFormOrSubFormById, selectRootForm } from '../reducers/form';
 
 export type RecordPickerProps = {
   disabled?: boolean;
-  recordId: string | undefined;
-  setRecordId: (recordId: string | undefined) => void;
+  recordId: string | null;
```

Comment:
```suggestion
  recordId?: string;
```

Reply:
the `?` accepts undefined right? I'd like to coerce to null, since that's what we gonna send from the api... Whatdya think?

Reply:
I see, in that case better leave it as is, but it means that we will have to explicitely declare the prop as `null` anytime there is no value, I case that's the behavior you want?

Reply:
I don't know, what do you think @internetti @neb42

Reply:
yeah, it is less convenient to explicitly declare null everytime, but if that's what comes from the api, that's what we'll have to do.
we could write mappers that could transform any api payload into a form more convenient for the frontend, if the need for that becomes high enough, but I don't see that yet.

Reply:
Isn't there something with the `<input>` that we need to pass it a `null` or a `string`, otherwise it complains about being controller/uncontrolled ?

Reply:
I'd say `string | null` because `recordId` should default to something and that should be either `null` or `string`. I don't see where it would be `undefined`